.gitignore

```diff
@@ -5,7 +5,7 @@ src/bitcoin
 src/zcashd
 src/zcash-cli
 src/zcash-gtest
-src/bitcoin-tx
+src/zcash-tx
 src/test/test_bitcoin
```

Comment:
It is not. However, I could look into renaming it for consistency; I only didn't because I expected it would require more time to ensure that e.g. the CI server handles the change (I *think* it should because it uses `full-test-suite.sh` but I'm not certain).